src/components/TeaserMyMagazine/TeaserMyMagazine.js

```diff
@@ -0,0 +1,302 @@
+                    {notificationsLabel}
+                  </TeaserSectionTitle>
+                </Link>
+              </div>
+              {latestSubscribedArticles.map(document => {
+                const {
+                  format,
+                  path,
+                  shortTitle,
+                  title,
+                  credits
+                } = document.meta
+
+                return (
+                  <TeaserFeed
+                    Link={Link}
+                    color={colorScheme.text}
+                    format={format}
+                    path={path}
+                    title={
+                      shortTitle
+                        ? `${shortTitle.substring(0, 140).trim()} ${
+                            shortTitle.length >= 140 ? '…' : ''
+                          }`
+                        : title
+                    }
+                    credits={credits}
+                  />
+                )
+              })}
+            </div>
+          ) : null}
+        </div>
+      </section>
+    </div>
+  )
+}
+
+const styles = {
+  section: css({
+    margin: '0 auto',
+    maxWidth: 1300,
+    padding: '40px 15px 10px',
+    [mUp]: {
+      padding: '50px 15px 40px'
+    }
+  }),
+  sectionTitle: css({
+    marginBottom: 24,
+    [mUp]: {
+      marginBottom: 36
+    }
+  }),
+  row: css({
+    display: 'flex',
+    flexDirection: 'column',
+    [mUp]: {
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'flex-start'
+    }
+  }),
+  left: css({
+    marginBottom: 32,
+    [mUp]: {
+      flex: 1,
+      marginRight: 16,
+      marginBottom: 0
+    }
+  }),
+  right: css({
+    [mUp]: {
+      flex: 1,
+      marginLeft: 16
+    }
+  }),
+  center: css({
+    marginBottom: 0,
+    [mUp]: {
+      width: '50%',
+      marginBottom: 0
+    }
+  }),
+  tile: css({
+    padding: '16px 8px 12px 8px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    marginBottom: 16,
+    ':last-child': {
+      marginBottom: 30,
+      [mUp]: {
+        marginBottom: 40
+      }
+    },
+    [mUp]: {
+      padding: '12px 8px'
+    }
+  }),
+  tileHeadline: css({
+    textDecoration: 'none',
+    cursor: 'pointer',
+    wordWrap: 'break-word',
+    width: '100%',
+    ...serifBold17,
+    lineHeight: '18px',
+    [mUp]: {
+      ...serifBold19,
+      lineHeight: '21px'
+    }
+  })
+}
+
+const WrappedTeaserMyMagazine = props => (
+  <ColorContext.Provider value={colors.negative}>
+    <TeaserMyMagazine {...props} />
+  </ColorContext.Provider>
+)
+
+export default WrappedTeaserMyMagazine
+
+TeaserMyMagazine.propTypes = {
+  latestSubscribedArticles: PropTypes.array,
+  latestProgressOrBookmarkedArticles: PropTypes.array
+}
+
+WrappedTeaserMyMagazine.data = {
+  config: {
+    options: ({ first = 2 }) => ({
+      variables: {
+        first: +first
+      }
```

Comment:
```suggestion
      variables: {
        first: +first
      },
      ssr: false
```
And could you please add the same to the options of `TeaserActiveDebates/ActiveDebates.js?